jax/_src/scipy/special.py

```diff
@@ -2412,6 +2411,8 @@ def _hyp1f1_serie(a, b, x):
   https://doi.org/10.48550/arXiv.1407.7786
   """
 
+  precision = jnp.finfo(x.dtype).eps * 10
```

Comment:
For float32, this is `1E-6`, meaning this PR reduces precision in the float32 case. Is there a good justification for that?

Reply:
I changed this to the `eps`value so this is ~1e-7 by default for float32. My understanding of `eps` is probably bad since I thought it could be considered as the “resolution” of the associated dtype. In any case, I think it is an overkill to go that low, I expect 0-roundings to happen before we actually reach the precision of 1e-8, but I must admit I didn't test it extensively